### DISCUSSION TITLE
Can't delete completely a detection

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#636] [Q&A]
Hello,

Yesterday I had 3 false detections. So I went to "Recordings", "By Date", "2022-10-30", and selected "Puffin des Anglais", one of the false detection, then erased the audio by clicking on the trash icon.

However today I realized that the name "Puffin des Anglais" is still there on the same date, but the audio file is not available anymore:

![7u0EyOP](https://user-images.githubusercontent.com/83965802/199135938-e028d615-2ed1-4b42-8b2a-30d317b1b1ab.png)

So I went to "tools", "Database maintenance", and selected "Database: birds.db" then the "select" button on the left side to open the birds.db.

I then selected the right day: 

![S2D2dPj](https://user-images.githubusercontent.com/83965802/199136566-96cf116f-0521-44c1-bcba-f2f1f6afff83.png)

And noticed the bird was there, so it was not deleted when I deleted the audio file. What could be the reason?

Thank you.

**Comment by blogdemoi** [comment]
Edit: I just tried to manually delete the entry "Puffin des Anglais" and received this error message: "attempt to write a readonly database". So I guess this is the reason.

**Reply by blogdemoi** [reply]
So, how can I make the database writeable again? 😕

**Reply by ehpersonal38** [reply]
It's just like any file, `chmod 777 ~/BirdNET-Pi/scripts/birds.db`

**Reply by blogdemoi** [reply]
Thank you, I didn't know that. It worked.

**Reply by ehpersonal38** [reply]
You're welcome! Yeah any permission issues can usually be solved with that one command.
